src/sass/app_2/_common/drawer/drawer.scss

```diff
@@ -109,11 +111,47 @@ hr {
                         padding: 14px 8px;
 
                         &.btn-close {
-                            width: 1.5rem;
-                            height: auto;
-                            align-self: center;
+                            border-left: 1px solid $COLOR_GRAY;
                         }
                     }
+                    .notifications-header {
+                        padding: 0 16px;
+                        text-transform: capitalize;
+                        font-size: 16px;
+
+                        h4 {
+                            margin: 0.5rem;
+                            font-weight: 400;
+                        }
+                    }
+                }
+                .no-notifications-container {
+                    position: absolute;
+                    top: 53px;
+                    bottom: 0;
+                    left: 0;
+                    right: 0;
+                    height: calc(100% - 53px);
```

Comment:
Since the height of the drawer header is fixed to `53px`, I thought it would be fine. No worries, I'll redo the  styling.

Reply:
Its header height is _currently_ `53px` only because of the padding set on `.icon` ([L#111](https://github.com/binary-com/binary-static/pull/4116/files#diff-e70f32c6d5d3b647647446c002524a40R111)) and also the close button's height. 😄 

That's why it's not easy to maintain, as if a developer updates that padding in the future, they have to update `53` in other places.

Reply:
Yes I agree, we have to change the way we handle padding with the icons. This should be sorted together when we refactor the way of handling our icon/asset library.